plugins/janus_videoroom.c

```diff
@@ -5500,7 +5500,7 @@ static json_t *janus_videoroom_process_synchronous_request(janus_videoroom_sessi
 				{
 					janus_mutex_lock(&participant->rec_mutex);
 					gboolean prev_recording_active = participant->recording_active;
-					participant->recording_active = recording_active;
+					participant->recording_active = TRUE;
```

Comment:
Why do you need it? This is used only for the "enable recording" request (which we don't use now)